build.sh

```diff
@@ -31,6 +31,7 @@ PREFIX_H="$PREFIX_BUILD/include/"
 
 PREFIX_ROOTFS="$PREFIX_FS/root/"
 : "${PREFIX_ROOTSKEL:="$PREFIX_PROJECT/_fs/root-skel/"}"
+ROOTFS_OVERLAYS="${PROJECT_ROOTFS_OVERLAY}"
```

Comment:
too late, makes it impossible to overwrite in target/project files (included by `build.project`)

Also: `PROJECT_ROOTFS_OVERLAY` is undefined?

Reply:
I would go for:
```bash
ROOTFS_OVERLAYS="$PROJECT_PATH/rootfs-overlay:${ROOTFS_OVERLAYS}"
```
(and maybe define `ROOTFS_OVERLAYS=""` with comment what it is and that can be overridden in project files - before `build.project` include)

so `rootfs-overlay` subdir in project would be applied by default (if exists, that's why I need the check "if exists" in the `for` loop below).

That way no additional changes in `build.project` / project files are needed ATM.